### PR TITLE
A start at module/class#to_s

### PR DESCRIPTION
**Pull request by wied03**

### CODE REVIEW

File: core/class/to_s_spec.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+describe "Class#to_s" do
+  it "works when name is defined" do
+    CoreClassSpecs::Record.to_s.should == "CoreClassSpecs::Record"
+  end
+  
+  it "works with an anonymous class" do
+    c = Class.new
+    c.to_s.should =~ /#<Class:0x[0-9a-f]+>/
+  end
+end
```
Comment (eregon) [inline core/class/to_s_spec.rb]: There is no `Class#to_s`, only `Module#to_s`:
```ruby
> String.method :to_s
=> #<Method: Class(Module)#to_s>
```

File: core/module/to_s_spec.rb [hunk 2]
```diff
@@ -2,5 +2,12 @@
 require File.expand_path('../fixtures/classes', __FILE__)
 
 describe "Module#to_s" do
-  it "needs to be reviewed for spec completeness"
+  it "works when name is defined" do
```
Comment (eregon) [inline core/module/to_s_spec.rb]: `returns the full constant path leading to the module`

File: core/class/to_s_spec.rb (hunk 1 above)
Comment (eregon) [inline core/class/to_s_spec.rb]: These examples could be merged with the ones in `core/module/to_s_spec.rb` of course.

**Comment by wied03** [comment]
I didn't know whether to put that in class or module, but consolidated it to module and incorporated your feedback

**Comment by eregon** [comment]
Looks great now, thanks!
Yes, I should detail the file organization better since it's quite complex (but `#method` alone is almost good enough to decide).

**Pull request merged**